app/components/ReportingRequirement/Status.tsx

```diff
@@ -11,9 +11,14 @@ const Status: React.FC<Props> = ({
   upcomingReportDueDate,
   reportSubmittedDates,
 }) => {
+  let reportString = "Reports ";
+  if (reportType === "Project Summary") {
+    reportString = "Report ";
+  }
+
```

Comment:
There will only ever be one Project Summary Report so I wanted the copy to reflect that.

Reply:
we try to use `const` where possible:

```suggestion
  const reportString = 
    reportType === "Project Summary" ? "Reports " : "Report ";
```